src/main/java/org/gridsuite/modification/server/repositories/ModificationRepository.java

```diff
@@ -48,4 +50,12 @@ public interface ModificationRepository extends JpaRepository<ModificationEntity
     Set<GeneratorCreationEntity> findAllCreationsWithReactiveCapabilityCurvePointsByIdIn(List<UUID> ids);
 
     Integer countByGroupIdAndStashed(UUID groupId, boolean stashed);
+
+    @Modifying
+    @Query("update ModificationEntity m set m.modificationsOrder = :order where m.id = :id")
+    int setOrderById(@Param("order") int order, @Param("id") UUID id);
+
+    @Modifying
+    @Query("update ModificationEntity m set m.group.id = :newGroup where m.id = :id")
+    int setGroupById(@Param("newGroup") UUID newGroup, @Param("id") UUID id);
```

Comment:
Did you try to modify modification list at group level and the referenced group at modification level, in entity objects directly ? So that JPA handles the DB changes directly

Reply:
I revert these changes: no need anymore